### PR TITLE
Implement library not found page

### LINKED ISSUE #78: Design library not found page(or other error page)
Current one is too simple ha ...

[Bountysource](https://www.bountysource.com/issues/26648386-design-library-not-found-page-or-other-error-page?utm_source=26648386&utm_medium=shield&utm_campaign=ISSUE_BADGE)

screenshot from 2015-09-13 15 27 28

### PR DESCRIPTION
**Pull request by dakshshah96**
close #78 

@PeterDaveHello As posted [here](https://github.com/cdnjs/new-website/issues/78#issuecomment-304511021), the content inside the template doesn't get displayed on encountering a 404 not found error.

### CODE REVIEW

File: templates/notfound.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="container">
+  <div class="row">
+  <div class="col-lg-12">
+      <h3>Sorry, this page isn't available</h3>
+      <p>The link you followed may be broken, or the page may have been removed.</p>
+      <p>If you believe this was an error, report the bug on our <a target="_blank" href="https://github.com/cdnjs/new-website/issues">issues page</a>.</p>
+      <p>In the meantime, try browsing through our menu up top or refreshing.</p>
+  </div>
+  </div>
+</div>
```
Comment (PeterDaveHello) [inline templates/notfound.html]: There should be a new line char here.

File: webServer.js [hunk 2]
```diff
@@ -107,7 +107,8 @@ function start() {
     about: getTemplate('templates/about.html'),
     tutorials: getTemplate('templates/tutorials.html', true),
     tutorial: getTemplate('templates/tutorial.html', true),
-    api: getTemplate('templates/api.html', true)
+    api: getTemplate('templates/api.html', true),
+    404: getTemplate('templates/notfound.html', true)
```
Comment (PeterDaveHello) [inline webServer.js]: `404` should be `notfound`

Comment (dakshshah96) [inline webServer.js]: Haha, such a silly error! Thanks 😄

**Comment by dakshshah96** [comment]
@PeterDaveHello Works like a charm now 👍

**Comment by PeterDaveHello** [comment]
Let's post a screenshot?

**Comment by dakshshah96** [comment]
@PeterDaveHello Here it goes 😄
screenshot 2017-06-02 15 12 53

File: templates/notfound.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <div class="row">
+  <div class="col-lg-12">
+      <h3>Sorry, this page isn't available</h3>
+      <p>The link you followed may be broken, or the page may have been removed.</p>
+      <p>If you believe this was an error, report the bug on our <a target="_blank" href="https://github.com/cdnjs/new-website/issues">issues page</a>.</p>
+      <p>In the meantime, try browsing through our menu up top or refreshing.</p>
```
Comment (PeterDaveHello) [inline templates/notfound.html]: Should cut down two spaces for correct indent?

File: webServer.js [hunk 4]
```diff
@@ -492,6 +493,16 @@ function start() {
     }));
   });
 
+  app.use(function(req, res) {
+    res.status(404).send(generatePage({
+      reqUrl: req.url,
+      page: {
+        template: templates.notfound,
+        title: '404 Page Not Found ' + TITLE
```
Comment (PeterDaveHello) [inline webServer.js]: The title seems not working in your screenshot?

**Comment by dakshshah96** [comment]
@PeterDaveHello Fixed both the problems 👍 

Also, I noticed that the title is not displayed correct for other pages as well (About, API). I'll send a separate pull request to correct those 😄

**Comment by PeterDaveHello** [comment]
@dakshshah96 awesome

**Pull request merged**